specs/order_spec.rb

```diff
@@ -79,32 +80,40 @@
 end
 
 # TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Order Wave 2" do
+describe "Order Wave 2" do
   describe "Order.all" do
     it "Returns an array of all orders" do
-      # TODO: Your test code here!
+      result = Grocery::Order.all
+      result.must_be_instance_of(Array)
```

Comment:
You should also be checking that the array has the expected length, and that each element in the array is an `Order`.